src/index.html

```diff
@@ -22,6 +22,373 @@
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list-items">
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              data-qa="nav-hover"
+              href="index.html"
+              class="nav__list-link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product photo"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__counter">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="/"
+          class="card__buy-button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
```

Comment:
Each card article has the same `data-qa="card"` attribute. This is fine for one card, but if you have multiple cards, you should consider using a more unique identifier for each card if they are going to be used for QA or testing purposes.